SETUP_LOCAL_CLUSTER.md

```diff
@@ -31,12 +31,18 @@ cd hoprnet-master
 make -j deps && make -j build
 ```
 
-1. **Run the one-line setup script**: Proceed to run the following script. If you are planning to run [MyneChat](http://app.myne.chat/)
-   alongside, then make sure to pass the `-m` flag with your MyneChat instance URL. Please wait while this script creates
-   the local blockchain network and deploys the project contracts. In average, the process can take between 2-6 minutes,
-   depending on your computer capacity. **Important**, make sure to have both `curl` and `jq` installed in your computer
-   before running the script, as both are used. Please be aware you also need a version of `bash` of `5.x` or superior,
-   which in most macOS devices require an upgrade, the easiest being via `brew bash`.
+2. **Run the one-line setup script**: Proceed to run the following script.
+
+```
+./scripts/setup-local-cluster.sh -i topologies/full_interconnected_cluster.sh --listen-host 0.0.0.0
+```
+
+**Important**, make sure to have both `curl` and `jq` installed in your computer before running the script, as both are used.
+Please be aware you also need a version of `bash` of `5.x` or superior, which in most macOS devices require an upgrade, the easiest being via `brew bash`.
```

Comment:
This should not be comments. Prerequisites should be checked by the script, something like this:
```bash
if ! command -v <COMMAND> &> /dev/null
then
    echo "<the_command> could not be found"
    exit 1
fi
```